apps/ledger-live-desktop/package.json

```diff
@@ -162,6 +162,7 @@
     "@jest/globals": "^29.7.0",
     "@ledgerhq/esbuild-utils": "workspace:*",
     "@ledgerhq/hw-transport-mocker": "workspace:^",
+    "@ledgerhq/live-cli": "link:../cli",
```

Comment:
Why `link` and not `workspace`?

Reply:
Because I had this error when I try to add with `pnpm desktop add -D  @ledgerhq/live-cli` :
>  ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@ledgerhq%2Fledger-key-ring-protocol: Not Found - 404
> This error happened while installing the dependencies of @ledgerhq/live-cli@24.7.2
> @ledgerhq/ledger-key-ring-protocol is not in the npm registry, or you have no permission to fetch it.
> No authorization header was set for the request.

But now, I changed it manually to `workspace:^` and it worked.